src/pages/index.js

```diff
@@ -66,29 +157,31 @@ constants.addNewCardButton.addEventListener("click", () => {
 });
```

Comment:
* You need to **`disable`** the submit button in the `card form` if the inputs are **empty**. It’s needed because I can click the button and add an empty card, but it should **not** happen. After the first submitting the button gets activated, and you need to **deactivate** it again. Even if I can't add an empty card, the button should look **disabled** (gray) anyway and have attribute `disabled`.


`resetValidation` should be called here on opening